lua/telescope/_extensions/file_browser.lua

```diff
@@ -85,15 +85,12 @@ local pconf = {
     },
   },
   on_input_filter_cb = function(prompt)
-    if prompt:sub(-1, -1) == os_sep then
+    if prompt == '~/' then
```

Comment:
I'm generally open to having such input callbacks as users can unset them, but we'd have to make sure, that they are reasonably OS-agnostic, as in entering `~/` or `/` should not trigger errors on Windows (should work as is on macOS, I presume).

Reply:
True - `/` would likely cause an issue on Windows - more evidence this should probably be a recipe